app/javascript/src/components/Dashboard/Notes/ListNote.jsx

```diff
@@ -2,54 +2,64 @@ import React from "react";
 import { Avatar, Dropdown } from "@bigbinary/neetoui/v2";
 
-const ListNote = ({ title, notesStatus, description, created_at, tag }) => (
-  <div>
-    <div className="" style={styles.boxStyle}>
-      <div style={styles.boxInner}>
-        <div>
-          <h4>{title}</h4>
-        </div>
-        <div>
-          <Dropdown
-            label=""
-            buttonStyle="text"
-            position="bottom-end"
-            icon={() => <MenuVertical size={18} />}
-          >
-            <li>Edit</li>
-            <li>Delete</li>
-          </Dropdown>
-        </div>
-      </div>
-      <p style={styles.descriptionStyles}>{description}</p>
-      <hr style={styles.lineStyle} />
-      <div className="mainBottom" style={styles.bottomSec}>
-        <div className="left" style={styles.leftSec}>
-          <button className="button" style={styles.button}>
-            {tag}
-          </button>
+const ListNote = ({
+  title,
+  notesStatus,
+  id,
+  description,
+  created_at,
+  tag,
+  handleDeleteNote
+}) => {
+  return (
+    <div>
+      <div className="" style={styles.boxStyle}>
+        <div style={styles.boxInner}>
+          <div>
+            <h4>{title}</h4>
+          </div>
+          <div>
+            <Dropdown
+              label=""
+              buttonStyle="text"
+              position="bottom-end"
+              icon={() => <MenuVertical size={18} />}
+            >
+              <li>Edit</li>
+              <li onClick={() => handleDeleteNote(id)}>Delete</li>
+            </Dropdown>
+          </div>
         </div>
-        <div className="right" style={styles.rightSec}>
-          <div className="icon" style={styles.icon}>
-            <Clock size={12} />
+        <p style={styles.descriptionStyles}>{description}</p>
+        <hr style={styles.lineStyle} />
+        <div className="mainBottom" style={styles.bottomSec}>
+          <div className="left" style={styles.leftSec}>
+            <button className="button" style={styles.button}>
+              {tag}
+            </button>
           </div>
-          <div
-            className="time"
-            style={styles.time}
-          >{`${notesStatus} ${created_at}`}</div>
-          <div className="avatar" style={styles.avatar}>
-            <Avatar
-              user={{
-                name: "neeto UI"
-              }}
-              size="small"
-            />
+          <div className="right" style={styles.rightSec}>
+            <div className="icon" style={styles.icon}>
+              <Clock size={12} />
+            </div>
+            <div
+              className="time"
+              style={styles.time}
+            >{`${notesStatus} ${created_at}`}</div>
+            <div className="avatar" style={styles.avatar}>
+              <Avatar
+                user={{
+                  name: "neeto UI"
+                }}
+                size="small"
+              />
+            </div>
           </div>
         </div>
       </div>
     </div>
```

Comment:
Please refrain from using `style={}`. At BigBinary we either use neetoUI or tailwindCSS or custom SCSS. 
Also, please split the ListNote component into a reusable component.

Reply:
Sure. Removed all the styles and now using tailwindCSS and neetoUI CSS.